src/components/AssetHeader/AssetHeader.tsx

```diff
@@ -63,7 +63,7 @@ export const AssetHeader: React.FC<AssetHeaderProps> = ({ assetId, accountId })
   const [view, setView] = useState(accountId ? View.Balance : View.Price)
   const [isLargerThanMd] = useMediaQuery(`(min-width: ${breakpoints['md']})`)
   const asset = useAppSelector(state => selectAssetByCAIP19(state, assetId))
-  const chainId = asset.caip2
+  const chainId = asset?.caip2
```

Comment:
Note: This will produce a `useWalletSupportsChain: unknown chain id undefined` in case `asset.caip2` property cannot be accessed but that's actually great for debugging purposes.

Reply:
I put this in because I was getting a crash when asset was undefined but I think it was related to another change I was tested. I can probably just remove this.